src/deploy/functions/release/fabricator.ts

```diff
@@ -21,6 +21,7 @@ import * as reporter from "./reporter";
 import * as run from "../../../gcp/run";
 import * as scheduler from "../../../gcp/cloudscheduler";
 import * as utils from "../../../utils";
+import * as authBlocking from "../services/authBlocking";
```

Comment:
Are we really going to have different files for auth and authBlocking?

Reply:
Good catch, I'll rename it